daliuge-engine/dlg/manager/proc_daemon.py

```diff
@@ -207,6 +211,19 @@ def startDIM(self, nodes):
         logger.info(
             "Started Data Island Drop Manager with PID %d" % (self._dim_proc.pid)
         )
+
+        # Registering the new DIM via zeroconf so it gets discovered
+        # by the Master Manager
+        if self._zeroconf:
+            addrs = utils.get_local_ip_addr()
+            logger.info("Registering this DIM with zeroconf: %s" % addrs)
```

Comment:
Here we need the opposite: the string formatting should be lazily evaluated by the logging system:

```suggestion
            logger.info("Registering this DIM with zeroconf: %s", addrs)
```